src/components/Sidebar.vue

```diff
@@ -1,56 +1,59 @@
 <template>
   <div
-    @mouseover="hover = true"
-    @mouseleave="hover = false"
-    :class="[hover ? 'sidebar' : 'sidebar collapsed-sidebar']"
+    @mouseover="expanded = true"
+    @mouseleave="expanded = false"
+    :class="[expanded ? 'sidebar expanded' : 'sidebar collapsed-sidebar']"
   >
-    <div :class="[hover ? 'brand' : 'brand collapsed-brand']">
-      <router-link data-test="logo-to-main" class="logo" to="/wallet/transactions">
-        <img v-if="hover" class="sheikah-img" src="@/resources/svg/sheikah.svg" />
-        <img v-else class="sheikah-img" src="@/resources/svg/sheikah-small.svg" />
-      </router-link>
-    </div>
+    <router-link
+      data-test="logo-to-main"
+      :class="[expanded ? 'brand expanded' : 'brand collapsed-brand']"
+      to="/wallet/transactions"
+    >
+      <img class="sheikah-logo" src="@/resources/svg/sheikah-small.svg" />
+      <img v-if="expanded" class="sheikah-name" src="@/resources/svg/sheikah.svg" />
+    </router-link>
     <div class="current-wallet">
-      <span
-        :class="[
-          hover ? 'current-wallet-name' : 'current-wallet-name collapsed-current-wallet-name',
-        ]"
-      >
-        My Wallet
-      </span>
+      <NetworkStatus
+        :windowWidth="windowWidth"
+        :expanded="expanded"
+        :status="status"
+        :node="node"
+        :network="network"
+        :lastBlock="block"
+      />
     </div>
     <div class="link-list">
       <router-link data-test="to-transactions" class="link" to="/wallet/transactions">
         <font-awesome-icon class="icon" icon="wallet" />
-        <span :class="[hover ? 'label' : 'label collapsed-label']">Wallet</span>
+        <span v-if="expanded" class="label">Transactions</span>
       </router-link>
 
       <router-link data-test="to-templates" class="link" to="/request/templates">
         <font-awesome-icon class="icon" icon="code" />
-        <span :class="[hover ? 'label' : 'label collapsed-label']">Data request</span>
+        <span v-if="expanded" class="label">Data request</span>
       </router-link>
 
       <router-link data-test="to-marketplace" class="link" to="/marketplace">
         <font-awesome-icon class="icon" icon="shopping-bag" />
-        <span :class="[hover ? 'label' : 'label collapsed-label']">Marketplace</span>
+        <span v-if="expanded" class="label">Marketplace</span>
       </router-link>
 
       <router-link data-test="to-community" class="link" to="/community">
         <font-awesome-icon class="icon" icon="users" />
-        <span :class="[hover ? 'label' : 'label collapsed-label']">Community</span>
+        <span v-if="expanded" class="label">Community</span>
```

Comment:
I wonder whether removing the `<span>` elements is necessary if the sidebar itself is being resized down when collapsed, so they will be hidden.

In any case, I think `v-show` could be better than `v-if` in this case as it doesn't cause the element to be mounted and unmounted from the DOM but only toggles the `display` CSS property.